### PR TITLE
Sort files and folders in the share path

### PR DESCRIPTION
**Pull request by BenjiSchlotter**
### Motivation and context
Currently the files and folders lists under share path in the task creation view are not sorted. This can be annoying to work with when creating tasks manually for a lot of folders. 

### How has this been tested?
Added a path as share path with a lot of sub folders and checked manually that those folders are sorted alphabetically.

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
~~- [ ] I have added tests to cover my changes~~
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)) -> have not found related issues.
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
~~- [ ] I have updated the license header for each file (see an example below)~~

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by BenjiSchlotter** [comment]
@bsekachev I am not sure if I should update this change as a patch or a minor version. I am also not sure if should mention this small change in the documentation. Do you have input regarding this?

**Comment by bsekachev** [comment]
@BenjiSchlotter 

Hi, please ``patch`` version of cvat-ui.
No need to mention it in the documentation.
Thx.

File: cvat-ui/src/components/file-manager/file-manager.tsx [hunk 1]
```diff
@@ -129,6 +129,8 @@ export default class FileManager extends React.PureComponent<Props, State> {
 
     private renderShareSelector(): JSX.Element {
         function renderTreeNodes(data: TreeNodeNormal[]): JSX.Element[] {
+            // sort alphabetically
+            data.sort((a, b) => (a.key > b.key) ? 1 : -1)
```
Comment (bsekachev) [inline cvat-ui/src/components/file-manager/file-manager.tsx:133]: Static linter shows the issue here: ``Arrow function used ambiguously with a conditional expression.eslintno-confusing-arrow``

Maybe use such kind of sorting?
``data.sort((a: TreeNodeNormal, b: TreeNodeNormal): number => a.key.localeCompare(b.key));``

Comment (BenjiSchlotter) [reply cvat-ui/src/components/file-manager/file-manager.tsx]: I changed it to your suggestion.

**Comment by bsekachev** [comment]
@BenjiSchlotter 

Thanks again for the contribution

**Pull request merged**